src/igvfd/schemas/changelogs/reference_file.md

```diff
@@ -1,9 +1,12 @@
 ## Changelog for *`reference_file.json`*
 
+
 ### Minor changes since schema version 4
 
 * Add `vcf` to `file_format`.
 * Add `variants` to `content_type`.
+* Add `bed`, `csv`, `dat`, `gaf`, `gds`, `obo`, `owl`, `pwm` and `xml` to `file_format`.
```

Comment:
If "PWM" is capitalized in the schema enum, then I think the changelog should match it.